### PR TITLE
docs: add comment about ordering of addresses in wasmplus

### PR DESCRIPTION
**Pull request by kokeshiM0chi**
## Description

closes: #96

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes. (not needed)
- [x] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`  (not needed)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -48,6 +48,7 @@
 * [\#54](https://github.com/Finschia/wasmd/pull/54) add documentation about errors (codespace and codes)
 * [\#92](https://github.com/Finschia/wasmd/pull/92) modify links in x/wasmplus README.md
 * [\#103](https://github.com/Finschia/wasmd/pull/103) add clarifications to the order of list response in query
+* [\#107](https://github.com/Finschia/wasmd/pull/107) Added comment about ordering of addresses
```
Comment (170210) [inline CHANGELOG.md:51]: add comment about ordering of addresses in wasmplus

**Comment by 170210** [comment]
And I think changing the title to 'docs: add...' would be better.

File: proto/lbm/wasm/v1/query.proto [hunk 2]
```diff
@@ -34,7 +34,7 @@ message QueryInactiveContractsRequest {
 // QueryInactiveContractsResponse is the response type for the
 // Query/InactiveContract RPC method.
 message QueryInactiveContractsResponse {
-  // addresses is the inactive address list
+  // addresses is the inactive address list of byte arrays ordered in ascending order
```
Comment (loloicci) [inline proto/lbm/wasm/v1/query.proto:37]: isn't it a list of strings?

Comment (loloicci) [reply proto/lbm/wasm/v1/query.proto:37]: how about 
... list ordered in ascending order of byte formatted addresses

Comment (kokeshiM0chi) [reply proto/lbm/wasm/v1/query.proto:37]: This means ascending order when expressed as a byte array, but the type is string.

Comment (loloicci) [reply proto/lbm/wasm/v1/query.proto:37]: this can be read as
"addresses is the inactive address list of byte arrays _which is_ ordered in ascending order

**Review by loloicci — APPROVED** [review]

**Review by 170210 — APPROVED** [review]

**Pull request merged**